api/tests/test_canteens.py

```diff
@@ -400,9 +400,9 @@ def test_patch_with_own_siret(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
     @authenticate
-    def test_refuse_patch_with_no_siret(self):
+    def test_refuse_patch_with_blank_siret(self):
```

Comment:
pour simplifier un peu il est possible aussi de regrouper ces 2 tests en 1 seul, en faisant une boucle for, genre `for SIRET_VALUE_NOT_OK in ["", None]`